teach_base/lesson_01/01_ideal_weight.rb

```diff
@@ -0,0 +1,18 @@
+=begin
```

Comment:
Refactor comments according to the style guide:
https://github.com/rubocop-hq/ruby-style-guide#no-block-comments